### DISCUSSION TITLE
ytix.xyz

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2094] [Q&A]
this is about ads on site 

### URL= `https://ytix.xyz/convert/43299/OGey74KvjIw`
 opera browser stable 
ubo 1.42.5b1

STR=

1. clear cache+cookie

2. add `ytix.xyz##+js(acis, parseInt, break;case $.)` or `ytix.xyz##+js(aeld, , break;case $.)`

3. navigate to  `https://ytix.xyz/convert/45846/OsNz-GMROY8`

4. site is loaded correctly with no popup ads

5. now click top upper left thumbnail named `ytix.xyz`

6. after loading THEN CLICK BACK,site is loaded with ads giving popups when clicked anywhere.....

is this because site loads from cache or inefficient filter?

**Comment by hello-smile6** [comment]
Is there a legitimate reason to enable JavaScript on that site?

**Comment by mapx-** [comment]
I can reproduce in chrome too

working => `||ytix.xyz^$csp=script-src *`

**Reply by ghajini** [reply]
yes i tried b4hand csp filter `||ytix.xyz^$csp=default-src 'unsafe-inline' 'self' *.gstatic.com *.googleapis.com *.ytimg.com
`,no popups....
your csp filter breaks download
But i do want to know about scriptlet

**Reply by mapx-** [reply]
test this 1, it seems working 

`ytix.xyz##+js(aost, String.prototype.charAt, inlineScript, 2)`

**Reply by mapx-** [reply]
hmm working in chrome, not in opera

**Reply by mapx-** [reply]
no ... failing too in chrome

**Reply by ghajini** [reply]
@gorhill @gwarser

**Reply by ghajini** [reply]
i noticed when console is open from start & do above STR , they didn't kick in
